src/index.html

```diff
@@ -1,17 +1,179 @@
 <!doctype html>
 <html lang="en">
-  <head>
+  <head> 
+    <title>document</title>
     <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    
+    <fieldset class="group_first_field">
+      <legend>Personal information</legend>
+      <div class="input_first_field">
+        <label for="surname_field">Surname:</label>
+        <input 
+            id="surname_field" 
+            type="text"
+            name="surname"
+            autocomplete="off"
+        >
+      </div>
```

Comment:
Apply code styling [recommendation](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) all over the project. Here i a point №3
```suggestion
        <label for="surname_field">Surname:</label>
        <input 
          id="surname_field" 
          type="text"
          name="surname"
          autocomplete="off"
        >
      </div>
```